src/components/FormElements/XdrLedgerKeyPicker.tsx

```diff
@@ -108,58 +110,61 @@ export const XdrLedgerKeyPicker = ({
 }: XdrLedgerKeyPicker) => {
   const [selectedLedgerKey, setSelectedLedgerKey] =
     useState<LedgerKeyFieldsType | null>(null);
-
   const [isXdrInputActive, setIsXdrInputActive] = useState(true);
   const [formError, setFormError] = useState<AnyObject>({});
-  const [ledgerKeyXdrJsonString, setLedgerKeyJsonString] = useState<string>("");
+  const [ledgerKeyXdrToJsonString, setLedgerKeyJsonString] =
```

Comment:
Should we follow the convention naming the set state method?
```suggestion
  const [ledgerKeyXdrToJsonString, setLedgerKeyXdrToJsonString] =
```